pkgs/desktops/gnome/extensions/EasyScreenCast/default.nix

```diff
@@ -26,6 +26,7 @@ stdenv.mkDerivation rec {
   makeFlags = [ "INSTALLBASE=$(out)/share/gnome-shell/extensions" ];
 
   uuid = "EasyScreenCast@iacopodeenosee.gmail.com";
```

Comment:
It should probably be okay to rename the attribute without keeping the old one. This is not really an API we promised to keep; Hyrum, or not.